TheDailyWtf/Views/Home/Index.cshtml

```diff
@@ -1,6 +1,7 @@
 ﻿@model HomeIndexViewModel
 @{
     Layout = "~/Views/Shared/_Layout.cshtml";
+    SidebarWtfs = Model.GetSideBarWtfs();

```

Comment:
Good question.  I'm trying to run this locally so I can test it there, definitely don't think it's worth deploying/reverting (though it's easy enough to do through the Github UI).